src/classes/UTIL_OrgTelemetry_SObject_BATCH_TEST.cls

```diff
@@ -36,26 +36,40 @@
 @IsTest(IsParallel=false)
 private class UTIL_OrgTelemetry_SObject_BATCH_TEST {
 
+    private static UTIL_FeatureManagement_TEST.Mock featureManagementMock = new UTIL_FeatureManagement_TEST.Mock();
+
     /**
      * @description Create test data needed for the unit tests in this class
      */
     @TestSetup
     static void testDataSetup() {
+        TDTM_Config_API.disableAllTriggers();
```

Comment:
With the FeatureManagement mock, I thought it be easier to set up the test data manually instead of relying on the triggers to create the opportunities so that it's easier to track the counts.